contents/blog/cheaper-analytics.md

```diff
@@ -0,0 +1,72 @@
+---
+date: 2024-03-21
+title: 'PostHog is now up to X cheaper'
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author:
+  - james-hawkins
+category: PostHog news
+tags:
+  - Product updates
+---
+
+## tl;dr
+
+* It is now possible to use PostHog for X less
+* This change applies to all existing and new customers
+* The average customer can save X%
+* This cuts pricing for product analytics and web analytics
+* We are intending on more price cuts, this is the start!
```

Comment:
We should consider if we want to announce this now, or not. If we imply it's going to get cheaper (whether via control or reduction) later then users may wait before signing up. 

It's also always possible that decisions will change. I'm all for being open about what we're working on, but in an announcement like this it's best to be clear and certain - focus on what is changing _now_, rather than what probably will change _later_.